packages/victory-native/src/helpers/wrap-core-component.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+/**
+ * Wrap a core component, pass props through.
+ * @param {any} Component Core Component
+ * @param {object} defaultProps Default props for component
+ * @returns {React.FC} WrappedComponent Wrapped component (passes props through)
+ */
+export const wrapCoreComponent = ({ Component, defaultProps }) => {
```

Comment:
This is a helper method that wraps up a core component, and sets default props accordingly. This is used instead of class extension to allow the conversion of class components to functional components.

Reply:
I generally like this approach. It breaks down for some of what we had been doing with class extension, (i.e. the containers), but we can look into other ways to avoid that pattern for those.